examples/portfolio/src/pages/about.astro

```diff
@@ -1,77 +1,130 @@
 ---
-import MainHead from '../components/MainHead.astro';
-import Footer from '../components/Footer.astro';
-import Nav from '../components/Nav.astro';
+import BaseLayout from '../layouts/BaseLayout.astro';
+
+import ContactCTA from '../components/ContactCTA.astro';
+import Hero from '../components/Hero.astro';
 ---
 
-<html lang="en">
-	<head>
-		<MainHead title="About | Jeanine White" description="About Jeanine White Lorem Ipsum" />
-		<style>
-			.heroImg {
-				max-height: 24rem;
-				object-fit: cover;
-				overflow: hidden;
-			}
+<BaseLayout title="About | Jeanine White" description="About Jeanine White Lorem Ipsum">
+	<div class="stack gap-20">
+		<main class="wrapper about">
+			<Hero
+				title="About"
+				tagline="Thanks for stopping by. Read below to learn more about myself and my background."
+			>
+				<img
+					width="1553"
+					height="873"
+					src="/assets/at-work.jpg"
+					alt="Jeanine White at work with a colleague"
+				/>
+			</Hero>
 
-			.heroImg img {
-				width: 100%;
-			}
+			<section>
+				<h2 class="section-title">Background</h2>
+				<div class="content">
+					<p>
+						Cream cheese say cheese stinking bishop. <a href="https://astro.build/">Astro</a> makes people
+						happy. Brie fondue hard cheese bocconcini feta camembert de normandie babybel airedale. Red
+						leicester swiss manchego mascarpone pepper jack airedale fromage frais ricotta. Cheese and
+						biscuits cauliflower cheese boursin.
```

Comment:
Lol, I know this is from the original template, but maybe basic Lorem is more approachable?